src/Cli/dotnet/commands/dotnet-package/PackageCommandParser.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal class PackageCommandParser
+    {
+        private const string DocsLink = "https://aka.ms/dotnet-package";
+
+        public static CliCommand GetCommand()
+        {
+            CliCommand command = new DocumentedCommand("package", DocsLink);
```

Comment:
https://github.com/dotnet/sdk/blob/250dfb2edbc95773c088d34cc2ad0b72d79a8112/src/Cli/dotnet/commands/dotnet-restore/RestoreCommandParser.cs#L60

RestoreCommandParses, passes another parameter (LocalizableStrings.AppFullName). I am not sure how the change in the behavior because of passing the additional parameter.